conbench/tests/entities/test_history.py

```diff
@@ -52,7 +52,7 @@ def _get_head_of_default(commit: Commit) -> Commit:
         13.435028842544385,
         1.0928748862317967,
         -2.1857497724635935,
-        -4.486538431438488,
+        -4.486481486904943,
```

Comment:
Hm. I wonder why this changed z-scores?

...Oh. Our [mocked data](https://github.com/conbench/conbench/blob/e1cab9b8ebffaba1c85e454b1b13de7f14bfcf02/conbench/tests/api/_fixtures.py#L257) is posting self-calculated stats. That's not what I expected it to be doing.